### PR TITLE
Validate input configuration against schema

### PR DESCRIPTION
**Pull request by HansKallekleiv**
Validate user input configuration against predefined [JSON schemas](http://json-schema.org/) for each function (QCForward).
Will raise ValidationError if the provided configuration is invalid. 
E.g. missing required keys or having wrong types.

PR not ready for merge, but mainly meant as an input for discussion around this type of functionality.

Another possibility with this type of validation is to provide end-users with html forms (, e.g. using https://github.com/rjsf-team/react-jsonschema-form, ) instead of YAML files

### CODE REVIEW

**Comment by jcrivenaes** [comment]
Sounds like a very good idea!

File: .github/workflows/fmu-tools.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['2.7', '3.6', '3.7', '3.8']
+        python-version: ['3.6', '3.7', '3.8']
```
Comment (jcrivenaes) [inline .github/workflows/fmu-tools.yml:20]: Should we skip 2.7 now? It makes life easier...

Comment (HansKallekleiv) [reply .github/workflows/fmu-tools.yml:20]: Yes, lets make life easier.

**Comment by HansKallekleiv** [comment]
Superseded by #53

**Pull request closed without merging**